src/Rule/In.php

```diff
@@ -28,30 +30,33 @@ final class In implements RuleWithOptionsInterface, SkipOnErrorInterface, WhenIn
     use SkipOnErrorTrait;
     use WhenTrait;
 
+    /**
+     * @param iterable $values A set of values to check against.
```

Comment:
Need to mention that subarrays are not supported and recommend to use `Subset` instead. Maybe in the class description too.